spec/requests/api/v1/payments_spec.rb

```diff
@@ -0,0 +1,275 @@
+# frozen_string_literal: true
+
+require 'swagger_helper'
+
+RSpec.describe 'payments API', type: :request do
+  let(:user_id) { build(:confirmed_user).id }
+  let(:agency_id) { create(:agency).id }
+  let(:business_id) { build(:business).id }
+  let(:site_id) { create(:site).id }
+  let!(:payment_params) do
+    {
+      "agency_id": agency_id,
+      "amount_cents": '123400',
+      "care_finished_on": '2020-06-01',
+      "care_started_on": '2020-01-01',
+      "discrepancy_cents": '7890',
+      "paid_on": '2020-07-07',
+      "site_id": site_id
+    }
+  end
+
```

Comment:
Could we make another set of params with no discrepancy and make sure we still get a 201 on create?